python/podio_class_generator.py

```diff
@@ -222,15 +229,38 @@ def _process_datatype(self, name, definition):
     self._fill_templates('Obj', datatype)
     self._fill_templates('Collection', datatype)
     self._fill_templates('CollectionData', datatype)
+    self._fill_templates('MutableStruct', datatype)
+    self._fill_templates('Constructor', datatype)
 
     if 'SIO' in self.io_handlers:
       self._fill_templates('SIOBlock', datatype)
 
+  def _preprocess_for_julia(self, datatype):
+    """Do the preprocessing that is necessary for Julia code generation"""
+    includes_jl, includes_jl_struct = set(), set()
+    for relation in datatype['OneToManyRelations'] + datatype['OneToOneRelations'] + datatype['VectorMembers']:
+      if self._needs_include(relation) and not relation.is_builtin:
+        includes_jl.add(self._build_include(relation.bare_type, julia=True, is_struct=True))
+        # if datatype['class'].bare_type != relation.bare_type:
+        #   includes_jl.add(self._build_include(relation.bare_type + 'Collection', julia=True))
+    for member in datatype['VectorMembers']:
+      if self._needs_include(member) and not member.is_builtin:
+        includes_jl_struct.add(self._build_include(member.bare_type, julia=True))
+    datatype['includes_jl']['constructor'].update((includes_jl))
+    datatype['includes_jl']['struct'].update((includes_jl_struct))
+
+  def _get_julia_params(self, datatype):
```

Comment:
```suggestion
  @staticmethod
  def _get_julia_params(datatype):
```
Easy fix for the pre-commit complaint